templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -198,7 +198,29 @@ objects:
           - type: basic
             enabled: ${ENABLE_BASIC_AUTH}
 
+- apiVersion: servicecatalog.k8s.io/v1alpha1
+  kind: Broker
+  # kind: ServiceBroker
+  metadata:
+    name: ansible-service-broker
+  spec:
+    url: ${ASB_SCHEME}://asb-1338-ansible-service-broker.${ROUTING_SUFFIX}
```

Comment:
why didn't we just make the entire url a variable? and default it to `https://asb-1338-ansible-service-broker.172.17.0.1.nip.io`? I must be missing some critical information.

Reply:
I did it this way to match up with `asb_scheme` and `openshift_routing_suffix` from catasb.